deploy/streisand-existing-cloud-server.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+#
+# Provision an existing cloud server.
+#
+# This requires an expanded extra-vars file specific to the provider type that
+# sets all of the values gathered by prompts in the interactive installation.
+# See the contents of global_vars/noninteractive for examples that can be copied
+# and modified.
+#
+# Usage:
+# streisand-existing-cloud-server \
+#   --ssh-user root \
+#   --ip-address 10.10.10.10 \
+#   --site-config path/to/digitalocean-site.yml
+#
+
+set -o errexit
+set -o nounset
+
+DIR="$( cd "$( dirname "$0" )" && pwd)"
+PROJECT_DIR="${DIR}/.."
+
+export DEFAULT_SITE_VARS="${PROJECT_DIR}/global_vars/default-site.yml"
+export GLOBAL_VARS="${PROJECT_DIR}/global_vars/vars.yml"
+
+# Include the check_ansible function from ansible_check.sh.
+# shellcheck source=util/source_validate_and_deploy.sh
+source "${PROJECT_DIR}/util/ansible_check.sh"
+check_ansible
+
+# --------------------------------------------------------------------------
+# Reading options.
+# --------------------------------------------------------------------------
+
+function usage () {
+  cat <<EOF
+Usage:
+$0 \\
+  --ssh-user root \\
+  --ip-address 10.10.10.10 \\
+  --site-config path/to/site.yml
+EOF
+}
+
+SSH_USER=""
+IP_ADDRESS=""
+SITE_VARS=""
+
+while [[ ${#} -gt 0 ]]; do
+  case "${1}" in
+    # Required.
+    --ip-address)    IP_ADDRESS="${2}"; shift;;
+    --site-config)   SITE_VARS="${2}"; shift;;
+    --ssh-user)      SSH_USER="${2}"; shift;;
+
+    # Utility.
+    -h|--help)       usage; exit 0;;
+    --)              break;;
+    -*)              echo "Unrecognized option ${1}"; usage; exit 1;;
+  esac
+
+  shift
+done
+
+# --------------------------------------------------------------------------
+# Fail if required options are not set.
+# --------------------------------------------------------------------------
+
+if [ -z "${IP_ADDRESS}" ] || [ -z "${SITE_VARS}" ] || [ -z "${SSH_USER}" ]; then
```

Comment:
Minor nit: I think if `${SSH_USER}` is empty/not provided we should just default to using `root`.